nano/node/cli.cpp

```diff
@@ -306,6 +313,45 @@ std::error_code nano::handle_node_options (boost::program_options::variables_map
 		node.node->store.peer_clear (transaction);
 		std::cerr << "Database peers are removed" << std::endl;
 	}
+	else if (vm.count ("confirmation_height_clear"))
+	{
+		boost::filesystem::path data_path = vm.count ("data_path") ? boost::filesystem::path (vm["data_path"].as<std::string> ()) : nano::working_path ();
+		inactive_node node (data_path);
+		auto account_it = vm.find ("account");
+		if (account_it != vm.cend ())
+		{
+			auto account_str = account_it->second.as<std::string> ();
+			nano::account account;
+			if (!account.decode_account (account_str))
+			{
+				nano::account_info account_info;
+				auto transaction (node.node->store.tx_begin_read ());
+				if (!node.node->store.account_get (transaction, account, account_info))
+				{
+					auto transaction (node.node->store.tx_begin_write ());
+					node.node->store.confirmation_height_clear (transaction, account, account_info);
+					std::cerr << "Confirmation height of account " << account_str << " is set to 0" << std::endl;
```

Comment:
std::cout ?

Reply:
I copied the other `*_clear` ones which used `std::cerr` as well, but I agree it doesn't seem correct, so I'm going to change those as well.